### ISSUE TITLE
Insufficient color contrast

### ISSUE THREAD

**Issue by AmyJuneH** [badcamp/badcamp-2019#4]
The color combination #0077a5/#444444 has a contrast ratio of 1.94, which is not sufficient. At this size, you will need a ratio of at least 4.5.

Relevant code:
<a href="#main-content" class="visually-hidden focusable"> Skip to main content </a>

**Comment by sean-e-dietrich** [comment]
Closing as old theme
